jumpstart_ui.layouts.yml

```diff
@@ -32,6 +32,7 @@ jumpstart_ui_two_column:
 # Three Column
 jumpstart_ui_three_column:
   <<: *defaults
+  class: '\Drupal\jumpstart_ui\Layouts\JumpstartUiThreeColumnLayout'
```

Comment:
I think it would be more appropriate to set the class for the one column layout. then there will be less `unset` going on in the two and three columns

Reply:
I'd like to expand this functionality into the 2 and 3 column options but haven't the bandwidth to do that yet. If you could overlook the extra class and the unsets until I can get to it I don't think it will cause harm.

Reply:
![oh really](https://media1.tenor.com/images/09aeffc123a727299bf67b4b70602c05/tenor.gif?itemid=7570332)

Reply:
For realz. Grids in grids can get a little zany so that is why it is TBD at the moment.